core-services/ci-secret-mirroring-controller/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+- stevekuznetsov
```

Comment:
is there a reason why not `dptp`?

Reply:
Also, maybe this could go into `core-services/supplemental-ci-images`? Will we have more config for `secret-mirroring-controller` over this IS?

Reply:
https://github.com/openshift/release/tree/master/core-services/supplemental-ci-images

Reply:
Done.